### ISSUE TITLE
editor/tests EXE crashes in both debug/release mode. - current version 23/Jun/2022

### ISSUE THREAD

**Issue by redagent101** [turanszkij/WickedEngine#469]
editor/tests EXE crashes in both debug/release mode.

![image](https://user-images.githubusercontent.com/15969299/175282295-806fe39e-e258-44bf-8e86-eb80c2a9dd94.png)

![image](https://user-images.githubusercontent.com/15969299/175282220-4c92e50a-f037-4a0e-8f93-cf8eb8a8e169.png)

**Comment by redagent101** [comment]
Yes, windows and gfx drivers all up-to-date.  I last was able to run wicked engine back in May.  I have moved since then and unable to run tests/editors in both debug/release.

[cid:0bfeb06b-6275-4ea6-91de-e455fc48831e]

[cid:48cefb97-c575-4521-8182-c72b3942601a]

KR
Michael
________________________________

**Comment by redagent101** [comment]
got new PC now intel i7 with RTX 3060 ti , no issues now.  maybe wickedengine no longer compatible with the 1060 I had

**Comment by turanszkij** [comment]
It's strange, I have tested on a GTX 1070 after you posted this problem and there is no problem on that. Although it is not exactly a 1060, it is a similar hardware architecture. I was also developing the engine on a 1050 but not tested it recently because I don't have it any longer.

You could try entering `debugdevice` into the Debugger startup arguments and see the output log window in visual studio, maybe there will be a DX12 error posted there.

**Comment by redagent101** [comment]
It's strange for sure, it worked on the 1060 back in May before I moved then didn't work.  Maybe my pc setup changed somewhere .  Have new i7 pc and rtx 3060 ti and is very fast , wicked engine is ....wicked .

BR
Mike

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by plemsoft** [comment]
@redagent101 A workaround for the 1060 DX12 crash was added yesterday , so this issue can be closed, btw are you that BOTR ? :)
